src/ci_providers/provider_circleci.js

```diff
@@ -38,7 +33,7 @@ function _getSlug (inputs) {
   if (envs.CIRCLE_PROJECT_REPONAME !== '') {
     slug = `${envs.CIRCLE_PROJECT_USERNAME}/${envs.CIRCLE_PROJECT_REPONAME}`
   } else {
-    slug = `${envs.CIRCLE_REPOSITORY_URL}.git`
+    slug = `${envs.CIRCLE_REPOSITORY_URL.split(":")[1].split('.git')[0]}`
```

Comment:
Will this handle https:// URLs as well?

Reply:
probably not, but this is how it's handled in the bash uploader currently